examples/conformer_energies/conformer_energies.py

```diff
@@ -0,0 +1,93 @@
+from openforcefield.utils import RDKitToolkitWrapper
+from rdkit.Chem import rdMolAlign
+import numpy as np
+from simtk import openmm, unit
+
+def main(filename):
+    # Load in the molecule and its conformers.
+    # Note that all conformers of the same molecule are loaded as separate Molecule objects
+    # If using a OFF Toolkit version before 0.7.0, loading SDFs through RDKit and OpenEye may provide 
+    # different behavior in some cases. So, here we force loading through RDKit to ensure the correct behavior
+    rdktkw = RDKitToolkitWrapper()
+    loaded_molecules = Molecule.from_file(filename, toolkit_registry=rdktkw)
+    # Collatate all conformers of the same molecule
+    # NOTE: This isn't necessary if you have already loaded or created multi-conformer molecules;
+    # it is just needed because our SDF reader does not automatically collapse conformers.
+    molecules = [loaded_molecules[0]]
+    for molecule in loaded_molecules[1:]:
+        if molecule == molecules[-1]:
+            for conformer in molecule.conformers:
+                molecules[-1].add_conformer(conformer)
+        else:
+            molecules.append(molecule)
+
+    n_molecules = len(molecules)
+    n_conformers = sum([mol.n_conformers for mol in molecules])
+    print(f'{n_molecules} unique molecule(s) loaded, with {n_conformers} total conformers')
+
+    # Load the openff-1.1.0 force field appropriate for vacuum calculations (without constraints)
+    from openforcefield.typing.engines.smirnoff import ForceField
+    forcefield = ForceField('openff_unconstrained-1.1.0.offxml')
+    # Loop over molecules and minimize each conformer
+    for molecule in molecules:
+        # If the molecule doesn't have a name, set mol.name to be the hill formula
+        if molecule.name == '':
+            molecule.name = Topology._networkx_to_hill_formula(molecule.to_networkx())
+            print('%s : %d conformers' % (molecule.name, molecule.n_conformers))
+            # Make a temporary copy of the molecule that we can update for each minimization
+        mol_copy = Molecule(molecule)
+        # Make an OpenFF Topology so we can parameterize the system
+        off_top = molecule.to_topology()
+        print(f"Parametrizing {molecule.name} (may take a moment to calculate charges)")
+        system = forcefield.create_openmm_system(off_top)
+        # Use OpenMM to compute initial and minimized energy for all conformers
+        integrator = openmm.VerletIntegrator(1*unit.femtoseconds)
+        platform = openmm.Platform.getPlatformByName('Reference')
+        omm_top = off_top.to_openmm()
+        simulation = openmm.app.Simulation(omm_top, system, integrator, platform)
+
+        # Print text header
+        print('Conformer         Initial PE         Minimized PE       RMS between initial and minimized conformer')
+        output = [['Conformer','Initial PE (kcal/mol)','Minimized PE (kcal/mol)','RMS between initial and minimized conformer (Angstrom)']]
```

Comment:
You might have the method return the conformer energies and then just iterate through them here in writing out a new SDF.